examples/go/tagging/gotag-fn.yml

```diff
@@ -0,0 +1,16 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: http://127.0.0.1:8080
+functions:
+  gotag-fn:
+    lang: golang-http
+    handler: ./gotag-fn
+    image: fgold/gotag-fn:0.1.0
+    environment:
+      write_debug: true
+      read_debug: true
+    secrets:
+      - gotag-vcconfig
```

Comment:
This does not match the instructions in the README where the secret name is `vcconfig` (faas-cli secret create **vcconfig** --from-file=vcconfig.toml --tls-no-verify). Please use the name we also have in other examples here, i.e. vcconfig for consistency.